hacktoberfest.adoc

```diff
@@ -3,7 +3,7 @@ layout: hacktoberfest
 title: Hacktoberfest
 permalink: /hacktoberfest/
 image:
-  path: https://wildfly-security.github.io/wildfly-elytron/assets/images/posts/hacktoberfest2022.png
+  path: https://prarthonapaul.github.io/wildfly-elytron/assets/images/posts/hacktoberfest2023.png
 ---
 :page-liquid:
 
```

Comment:
@PrarthonaPaul Just a minor for line 21 regarding the tshirts. it seems they do not do that this year, based on a comment here: https://github.com/wildfly/wildfly.org/pull/459#discussion_r1337604715 so we should update that